models/gc_tiger_lb2/utils/TigerLB2Runner.py

```diff
@@ -0,0 +1,59 @@
+"""
+------------------------------------------------
+Mhub / DIAG - Run Module for Tiger LB2 Algorithm
+------------------------------------------------
+
+------------------------------------------------
+Author: Sil van de Leemput
+Email:  sil.vandeleemput@radboudumc.nl
+------------------------------------------------
+"""
+from mhubio.core import Instance, InstanceData, IO, Module, ValueOutput, Meta, DataType, FileType
+
+from pathlib import Path
+import numpy as np
+import SimpleITK as sitk
+import torch
+
+import sys
+import json
+
+
+@ValueOutput.Name('til_score')
+@ValueOutput.Meta(Meta(key="value"))
```

Comment:
This is a placeholder.
Consider adding valuable meta-data such as the value format (e.g., scale=percent, min=0, max=100)

Reply:
Ok, modified.